packages/react-aria-components/docs/Select.mdx

```diff
@@ -1045,6 +1045,37 @@ A `FieldError` can be targeted with the `.react-aria-FieldError` CSS selector, o
 
 <StateTable properties={docs.exports.FieldErrorRenderProps.properties} />
 
+
+### Custom select value
+
+Select passes the rendered children of the selected item in the render props of the `SelectValue` as well as if the placeholder should be showing. You can use this to Customize the value displayed in the Select at all times.
```

Comment:
Since this is specifically referencing the render props of SelectValue, what do you think about moving it under the SelectValue section specifically so it is easy to see the full list of render props available?

Reply:
I'd be fine with that, good suggestion

Reply:
you ok with this change? https://github.com/adobe/react-spectrum/pull/6425#discussion_r1645166855